js/client-teambuilder.js

```diff
@@ -1245,6 +1254,39 @@
 			this.save();
 		},
 
+		setNotesBlur: function (e) {
+			var textarea = $(e.currentTarget);
+			if (!$.trim(textarea.val())) textarea.parent().hide();
+			if (textarea.val().indexOf('|') >= 0 || textarea.val().indexOf(']') >= 0 || textarea.val().indexOf('\t') >= 0) {
+				app.addPopupMessage("Unfortunately, set notes can't contain tabs or the characters | and ], since they're used for storing teams.");
```

Comment:
I also think a fivefold increase in localStorage size is highly unreasonable to expect. That would require like 2300 characters worth of set notes for every single team. A doubling in size sounds more likely, and that'd only be the case for people annotating every team.
Anyway, why not just make the teambuilder warn to back up teams once a certain size is reached? I'd rather have people be able to choose to risk a possible expiration than deny it to everyone.